test/runner/lib/core_ci.py

```diff
@@ -309,6 +310,7 @@ def _start(self, auth):
 
         display.info('Started %s/%s instance %s.' % (self.platform, self.version, self.instance_id),
                      verbosity=1)
+        display.info('Instance %s/%s on %s' % (self.platform, self.version, self._uri), verbosity=1)
```

Comment:
Same thing here about using a single log entry instead of two. Use the same format as suggested above.